.github/ISSUE_TEMPLATE/05-documentation.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
I'd change the file name to `01.5-documentation.md`. Thanks to that the Documentation template will be listed right after the "Handsontable".

Reply:
Or ... maybe it'd be better to rename other files with some gaps (for future templates) for instance:
![image](https://user-images.githubusercontent.com/571316/122191030-a85ac480-ce92-11eb-9261-5e342697c7d4.png)?

Reply:
We also can rename them each time a new one is created. Those file are not that essential as source code

Good idea with putting as second item on the list 👍